site/src/examples/date-picker/SingleWithCustomPanel.tsx

```diff
@@ -14,13 +14,13 @@ import {
   DatePickerSingleInput,
   type SingleDateSelection,
   formatDate,
-  getCurrentLocale,
+  getCurrentLocale, SingleDatePickerError,
 } from "@salt-ds/lab";
 import { CustomDatePickerPanel } from "@salt-ds/lab/stories/date-picker/CustomDatePickerPanel";
```

Comment:
SO 92803, these aren't exports, maybe do something like https://github.com/jpmorganchase/salt-ds/blob/0c3721a43095c759f15398c986727179b689275d/site/src/examples/ag-grid-theme/CheckboxSelection.tsx#L2-L3

Reply:
will pickup in site docs updates